### PR TITLE
Update inja v3.4.0

### PR DESCRIPTION
**Pull request by MarcusMogg**
inja switch to c++17 after v3.4.0 https://github.com/pantor/inja/releases/tag/v3.4.0, that will cause `add_requires("inja master")`

so fix it

### CODE REVIEW

**Comment by MarcusMogg** [comment]
reopen for workflow (

**Comment by MarcusMogg** [comment]
Well, the FreeBSD workflow is functioning normally on my test branch without code change， https://github.com/MarcusMogg/xmake-repo/actions/workflows/freebsd.yml

File: packages/i/inja/xmake.lua [hunk 1]
```diff
@@ -15,6 +16,10 @@ package("inja")
     end)
 
     on_test(function (package)
+        local cxx_std = "c++11"
+        if package:version() == nil or package:version():satisfies(">=3.4.0") then
+            cxx_std = "c++17"
+        end
```
Comment (waruqi) [inline packages/i/inja/xmake.lua:22]:
```
local cxx_std = "c++17"
if package:version() and package:version():le("3.4.0") then
        cxx_std = "c++11"
end
```

Comment (MarcusMogg) [reply packages/i/inja/xmake.lua:22]: cool , the development documentation should explain the usage of semver versioning

**Pull request merged**